app/src/main/java/fr/free/nrw/commons/Utils.java

```diff
@@ -243,4 +244,11 @@ public static String getWLMEndDate() {
         return "30 Sep";
     }
 
+    public static int getWikiLovesMonumentsYear(Calendar calendar) {
```

Comment:
Would you mind writing a unit test for this? Thanks a lot!

Reply:
Done! https://github.com/commons-app/apps-android-commons/pull/5049/commits/d388a8d17ef44794aec475eba4f27bb82523c2bd